common/djangoapps/student/views.py

```diff
@@ -350,6 +352,22 @@ def _cert_info(user, course, cert_status):
         else:
             status_dict['download_url'] = cert_status['download_url']
 
+            # getting linkedin URL and then pass the params which appears
+            # on user profile. if linkedin config is empty don't show the button.
+
+            current_config = LinkedInUrlConfiguration.current()
+            if current_config.enabled:
+                cert_name = u'{type} Certificate for {cert_name}'.format(
+                    type=cert_status["mode"].title(), cert_name=course.display_name
```

Comment:
In the database, it looks like `title` is includes the word "Certificate" (for example: "Verified Certificate").  I'd prefer not to rely on this, though.  It would be safer to check the mode's `slug` field:

* "honor" --> "Honor Code Certificate"
* "verified" --> "Verified Certificate"
* "professional" --> "Professional Certificate"
* Anything else --> "Certificate"

Also, nitpick: `cert_name` here looks like it should be called `course_name`.